helpers/github_utils.py

```diff
@@ -0,0 +1,2018 @@
+"""
+Import as:
+
+import helpers.github_utils as hgitutil
+"""
+
+import datetime
+import functools
+import hashlib
+import itertools
+import json
+import logging
+import os
+import time
+from typing import Any, Callable, Dict, List, Literal, Optional, Tuple
+
+import github
+import IPython
+import matplotlib.pyplot as plt
+import pandas as pd
+import tqdm as td
+from tqdm import tqdm
+
+import helpers.hdbg as hdbg
+
+_LOG = logging.getLogger(__name__)
+
+
+# #############################################################################
+# GitHubCache
+# #############################################################################
+
+
+class GitHubCache:
```

Comment:
Probably SimpleCache would have been sufficient.

Can you take a look and see if we can reuse it? If not, let's add a TODO.

We want to reuse the code as much as possible to avoid reinventing the wheel.
That code is well tested and used everywhere